playground/nextjs-app-router/app/globals.css

```diff
@@ -2,6 +2,8 @@
 @tailwind components;
 @tailwind utilities;
 
+@import url('https://fonts.googleapis.com/css2?family=DM+Sans:wght@400;500;700&display=swap');
```

Comment:
We can create an `index -with-tailwind.css` file for all css imports. Similar to what we do in src